projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -142,21 +163,34 @@ export class OAuthLibWrapperService {
           take(1)
         )
         .subscribe((event) => (tokenReceivedEvent = event));
-
-      this.oAuthService
-        .tryLogin({
-          // We don't load discovery document, because it doesn't contain revoke endpoint information
-          disableOAuth2StateCheck: true,
-        })
-        .then((result: boolean) => {
-          resolve({
-            result: result,
-            tokenReceived: !!tokenReceivedEvent,
+      if (this.currentBaseSite.cdcSiteConfig) {
```

Comment:
if baseSite has cdcSiteConfig, then trylogin() need to load discovery document

Reply:
During each time a page is opened, this method will be called by AuthService to log in user. After logging successfully, the page will be redirected.
https://github.com/SAP/spartacus/pull/19144/files#diff-f5970a8646b27e360071eded1b6a7614f2691bd271f329e371b4137cce332581R52